packages/routes/src/RouteTest.tsx

```diff
@@ -22,8 +23,12 @@ export abstract class RouteTest extends BasicTest {
    * @param params Params object passed through TODO: think body vs query vs path params...
    */
 
-  async invoke(method: HttpMethod = 'GET', path = '/', params: any = {}) {
-    const { request, response } = this.requestAndResponse(method, path, params)
+  async invoke(path = '/', params: any = {}) {
+    const { request, response } = this.requestAndResponse(
+      this.route.method,
+      path,
+      params
+    )
```

Comment:
I suggest the following for improved readability
```javascript
    const method = this.route.method
    const { request, response } = this.requestAndResponse(method, path, params)
```

Reply:
or
```javascript
    const reqRes = this.requestAndResponse(this.route.method, path, params)
    await this.executeRoute(reqRes)
    return this.gatherResponseData(reqRes.response)
```

Reply:
Yes, this will theoretically be a breaking change, but unicorn is not using this until now. The original idea has been to allow routes to handle multiple http methods, but this is conceptually not possible with the current implementation, so to pass this as param doesn't make sense right now, it's just redundant to the static property.

Reply:
Yes, i was thinking about writing a helloworld example as well, will do that now.

Reply:
This was the answer to the question above - but didn't resloved my suggestion regarding the topic here

Reply:
resolved by https://github.com/seagull-js/seagull/pull/89/commits/dc6d2e111901d98975b003b36ccda661c512c69d